es_ES/animations/overview.md

```diff
@@ -0,0 +1,157 @@
+
+Por sí mismo, una Tween sólo define cómo interpolar entre dos valores. Para obtener un valor concreto para el frame actual de una animación, también necesita una animación para determinar el estado actual. Hay dos maneras de combinar un Tween con una animación para obtener un valor concreto:
+
+1. Puedes [`evaluar`](https://docs.flutter.io/flutter/animation/Tween/evaluate.html)
+el Tween en el valor actual de una animación. Este enfoque es muy útil para los widgets que ya están escuchando la animación y por lo tanto reconstruyendo cada vez que la animación cambia de valor.
+
+2. Puedes [`animar`](https://docs.flutter.io/flutter/animation/Animatable/animate.html)
+   el tween basado en la animación. En lugar de devolver un único valor, la función animada devuelve una nueva `Animación` que incorpora el tween. Este enfoque es más útil cuando se desea dar la animación recién creada a otro widget, que puede leer el valor actual que incorpora el tween así como escuchar los cambios en el valor.
+
+# Arquitectura
+
+Las animaciones se construyen a partir de una serie de elementos básicos.
+
+## Scheduler
+
+El
+[`SchedulerBinding`](https://docs.flutter.io/flutter/scheduler/SchedulerBinding-class.html)
+es una clase singleton que expone los primitivos de programación de Flutter.
+
+Para esta discusión, la clave primitiva son los frame callbacks. Cada vez que es necesario mostrar un cuadro en la pantalla, el motor de Flutter activa una llamada de retorno de "cuadro inicial" que el programador multiplexa a todos los oyentes registrados utilizando [`scheduleFrameCallback()`](https://docs.flutter.io/flutter/scheduler/SchedulerBinding/scheduleFrameCallback.html).
+Todas estas llamadas de retorno reciben el sello de tiempo oficial del frame, en forma de una `duración` de alguna parte temporal arbitraria. Dado que todas las llamadas de retorno tienen el mismo tiempo, cualquier animación activada desde estas llamadas de retorno parecerá estar exactamente sincronizada incluso si tardan unos pocos milisegundos en ejecutarse.
+
+## Tickers
+
+La clase [`Ticker`](https://docs.flutter.io/flutter/scheduler/Ticker-class.html) se conecta al mecanismo [`scheduleFrameCallback()`](https://docs.flutter.io/flutter/scheduler/SchedulerBinding/scheduleFrameCallback.html) del scheduler para invocar un callback de tick. 
+
+Un `Ticker` puede iniciarse y detenerse. Cuando se inicia, devuelve un `Future` que se resolverá cuando se detenga.
+
+Cada tick, el `Ticker` proporciona un callback con la duración desde el primer tick después de que fue iniciado.
+
+Debido a que los tick siempre dan su tiempo transcurrido en relación con el primer tick después de que se iniciaron, todos los ticks están sincronizados. Si inicia tres ticks en diferentes momentos entre dos frames, todos ellos se sincronizarán con la misma hora de inicio y, a continuación, se marcarán al mismo tiempo y posteriormente se tick en paralelo.
```

Comment:
If you start three ticks at different times between two frames, they will all nonetheless be synchronised with the same starting time, and will subsequently tick in lockstep.
Si inicias tres ticks en diferentes momentos entre dos frames, todos ellos se sincronizarán con el mismo tiempo de inicio y posteriormente marcarán el tick de bloqueo.

No comprendo el final de esta frase... que significa realmente "tick in lockstep"

Reply:
esta parte aun esta pendiente, como quedaría?

Reply:
Si se inician tres ticks en diferentes momentos entre dos frames, todos ellos se sincronizarán con la misma hora de inicio, y posteriormente se marcarán en forma sincronizada.